carbon-design-system/carbon-addons-iot-react
# feat(header): added ability to update aria-labels via i18n props

Closes #1725

**Summary**

- Adds an i18n dictionary to handle the main header and open menu aria labels

**Change List (commits, features, bugs, etc)**

- added i18n dictionary to Header.jsx
- added sample prop to Header.story.jsx

**Acceptance Test (how to verify the PR)**

- Observe Header.jsx to see the default i18n props
- check the dom in ?path=/story/watson-iot-header--header-action-buttons-with-dropdowns
- Observe 'Open menu' is overwritten with 'Open the menu'
